lib/Runtime/PlatformAgnostic/Platform/Common/Intl.cpp

```diff
@@ -131,6 +151,8 @@ class Output
         }                                                                     \
     } while (false)
 
+#define ASSERT_ENUM(T, e) Assert((int)(e) >= 0 && (e) < T::Max)
```

Comment:
>ASSERT_ENUM(T, e) Assert((int)(e) >= 0 && (e) < T::Max) [](start = 8, length = 55)

Consider what happens in release builds if this assert fails?

Reply:
I would suggest similarly here that since we control all of the call sites, its not really this code's responsibility to understand how to handle a bad enum value. If bad enums are being produced, thats a problem in our own code that should be able to be caught in debug builds.